src/style.js

```diff
@@ -238,6 +254,10 @@ const style = css`
     margin-top: auto;
     width: 100%;
   }
+  .sections .graph {
```

Comment:
Here, too: if this is useful also in non-section views, let's consider to just add it to the regular styles.

Reply:
I think this is only useful for section view. It makes graph position itself so that it's not clipping out of bounds.